app/views/groups/_group.html.erb

```diff
@@ -0,0 +1,8 @@
+<div class="groups">
+  <ul>
```

Comment:
With this partial you will have output that would reproduce ul and lis multiple times consider moving this outside of the partial. same with the div